test/PerpetualPositionRouter.t.sol

```diff
@@ -143,59 +143,88 @@ contract Fallback is PositionRouterTest {
       address(factory.computeAddress(PerpetualRouterFactory.RouterType.PositionRouterType, VETH));
   }
 
+  function helper_native_openPosition(
+    address token,
+    bool isBaseToQuote,
+    bool isExactInput,
+    uint256 amount,
+    uint256 oppositeAmountBound,
+    uint256 deadline,
+    uint160 sqrtPriceLimitX96
+  ) internal {
+    clearingHouse.openPosition(
+      IClearingHouse.OpenPositionParams({
+        baseToken: token,
+        isBaseToQuote: isBaseToQuote,
+        isExactInput: isExactInput,
+        amount: amount,
+        oppositeAmountBound: oppositeAmountBound,
+        deadline: deadline,
+        sqrtPriceLimitX96: sqrtPriceLimitX96,
+        referralCode: bytes32("")
+      })
+    );
+  }
+
   function testFork_OpenLongExactInputPosition() public {
+    uint256 snapshotId = vm.snapshot();
```

Comment:
IIUC we use the snapshot and revert because we open the same position twice: once with our router, and once directly, then we check they are equal? if so that makes sense, just confirming my read of this

Reply:
Yes, exactly